### DISCUSSION TITLE
`R` keyboard shortcut is broken

### DISCUSSION THREAD

**Discussion by mramato** [community/community#67519] [Accessibility] [answered]
### Select Topic Area

Bug

### Body

The ability to select text and press `R` seems to have broken some time over the last 2 weeks. It no longer quotes the selected text and instead quotes the entire comment. This feature is documented here: https://docs.github.com/en/get-started/using-github/keyboard-shortcuts so I assume this is a bug and not intentional.

I'm using Chrome Version 117.0.5938.88 (Official Build) (64-bit) on Kubuntu Linux 22.04, but I believe this is a general bug and not platform/browser specific.

**Comment by mramato** [comment, marked as answer]
This appears to have been fixed.

**Reply by ghostinhershell** [reply]
Good to know, thanks for confirming that this issue has been resolved on your end!
